### ISSUE TITLE
the abi-decoder.js is missing

### ISSUE THREAD

**Issue by brlojam4932** [ConsenSys/abi-decoder#70]
I am new to programing but this file is not there at all - there is no actual file named abi-decoder.js ...?
Am I missing something?

**Comment by PlanetTheCloud** [comment]
I have the same issue as well. Do you manage to find it?

**Comment by brlojam4932** [comment]
Yeah, someone posted on the comments that he probably purposely leaves out a piece from the lesson so that you will be more inclined to sign up to his services. It seems believable since this happens in most of his tutorials. I can never finish some of his tutorials. I think it's worth it to sign up the his coding bootcamps but I'm already on one.

**Comment by PlanetTheCloud** [comment]
Well, to those who wanted to decode the ABI, please take a look at https://web3js.readthedocs.io/en/v1.5.2/web3-eth-abi.html#decodeparameters. If you are trying to decode input data with a function call, remove the first 8 bytes of the HEX (https://github.com/ChainSafe/web3.js/issues/3016#issuecomment-828274902)

**Comment by Yiwen-Gao** [comment]
I think the code is all there in `index.js`, but it delegates some functionality to `abiCoder` from web3js.

https://github.com/ConsenSys/abi-decoder/blob/a9ed4923433ad32069e2ea963d683ed2ff49fa07/index.js#L2
